cookbooks/aws-parallelcluster-install/resources/lustre/partial/_install_lustre_debian.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :install_lustre do
+  apt_repository 'fsxlustreclientrepo' do
+    uri          node['cluster']['lustre']['base_url']
+    components   ['main']
+    key          node['cluster']['lustre']['public_key']
+    retries 3
+    retry_delay 5
+  end
+
+  apt_update
+
+  package "lustre-client-modules-#{node['cluster']['kernel_release']}" do
+    retries 3
+    retry_delay 5
+  end unless virtualized?
```

Comment:
Same as above.

Reply:
This is slightly different from the above case, as docker has a `kernel_release` value which is `5-11...-linux..` which does not map to an actual package. I could have added some if to force a specific version when on docker, but seemed overkill to me

Reply:
Could we use `KERNEL_RELEASE` from ubuntu AMIs?

Reply:
This should be fixed